dagobah/daemon/dagobahd.yml

```diff
@@ -17,9 +17,8 @@ Dagobahd:
 
   # choose one of the available backends
   # None: Dagobah will not use a backend to permanently store data
-  # sqlite: use a SQLite database. see the SQLite section in this file
   # mongo: store results in MongoDB. see the MongoBackend section in this file
-  backend: sqlite
+  backend: mongo
```

Comment:
Probably `None` so people can immediately get up and running to play around with it